CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/utilities/NewNotePane.java

```diff
@@ -123,43 +184,82 @@ public NewNotePane(final String userChosenColour) {
         cancelButton.setTextFill(Color.BLACK);
-        cancelButton.setOnAction(event -> closePopUp());
+        cancelButton.setOnAction(event -> {
+            clearTextFields();
+            closePopUp();
+        });
         cancelButton.setOnMouseEntered(event -> cancelButton.setStyle("-fx-background-color: #DBA800; "));
         cancelButton.setOnMouseExited(event -> cancelButton.setStyle("-fx-background-color: #DEC20B;  "));
-
         final Region gap = new Region();
         gap.setMinWidth(15);
         final HBox noteHBox = new HBox(5, newNoteColour, gap, addButton, cancelButton);
         HBox.setHgrow(gap, Priority.ALWAYS);
         noteHBox.setAlignment(Pos.CENTER_RIGHT);
-        final VBox addNoteVBox = new VBox(5, titleField, contentField, applyToSelection, noteHBox);
+
+        final HBox cbHBox = new HBox(30, applyToSelection, enableMarkdown);
+        final VBox addNoteVBox = new VBox(5, tabPane, cbHBox, noteHBox);
         addNoteVBox.setAlignment(Pos.CENTER_LEFT);
         addNoteVBox.setStyle(fontStyle + "-fx-padding: 5px;");
         addNoteVBox.setMinHeight(220);
 
         dialogPane.setBackground(new Background(new BackgroundFill(ConstellationColor.fromHtmlColor("#222222").getJavaFXColor(), null, null)));
 
         dialogPane.getChildren().add(addNoteVBox);
+
+        inEditMode.addListener((obj, old, newVal) -> {
+            noteHBox.getChildren().clear();
+            cbHBox.getChildren().clear();
+            if (newVal) {
+                noteHBox.getChildren().addAll(newNoteColour, gap, saveButton, cancelButton);
+                cbHBox.getChildren().add(enableMarkdown);
+            } else {
+                noteHBox.getChildren().addAll(newNoteColour, gap, addButton, cancelButton);
+                cbHBox.getChildren().addAll(applyToSelection, enableMarkdown);
+            }
+        });
+
+    }
+
+    private void resizeTextFlows(final TextFlow textFlow, final double scale) {
+        for (int i = 0; i < textFlow.getChildren().size(); ++i) {
+            if (textFlow.getChildren().get(i) instanceof TextFlow) {
+                resizeTextFlows((TextFlow) textFlow.getChildren().get(i), scale + 0.5);
+            }
+        }
+        textFlow.setMaxHeight(textFlow.getMaxHeight() / scale);
+        textFlow.setMinWidth(495);
     }
 
     /**
      * Instantiate stage for the pop up and set event handler to close it when
      * consty closes
      */
-    public void showPopUp(final Window window) {
+    public void showPopUp() {
         if (isFirstTime) {
             parent.setOnCloseRequest(event -> {
+                clearTextFields();
                 addButton.setDisable(true);
                 cancelButton.setDisable(true);
                 closePopUp();
             });
             stage = new Stage();
-            stage.initModality(Modality.WINDOW_MODAL);
+
+            stage.setOnCloseRequest(event -> clearTextFields());
+
+            stage.initModality(Modality.APPLICATION_MODAL);
```

Comment:
There is an issue with the modality of the Create/Edit Note dialog. You can still click on Menu items in the Netbeans environment, and can select different graph tabs, which causes strange/inconsistent behaviour in the Create/Edit Note window.